pdfminer/high_level.py

```diff
@@ -13,6 +13,28 @@
 from .pdfpage import PDFPage
 
 
+class open_filename(object):
```

Comment:
I agree that this should be moved to utils.py. If the current content of utils.py bothers you, perhaps we can move some of it to a better place in another issue/pr.